crates/bevy_window/src/window.rs

```diff
@@ -58,14 +58,27 @@ impl WindowRef {
     }
 }
 
-impl MapEntities for WindowRef {
-    fn map_entities<M: EntityMapper>(&mut self, entity_mapper: &mut M) {
+impl<'a> IntoIterator for &'a WindowRef {
```

Comment:
Having `IntoIterator ` for `WindowRef` is a bit weird... Maybe derive `InterEntities`?

Reply:
Not derivable currently (because enum), but it can be implemented manually. Or we can just do `MapEntities` if iteration feels weird as well.

Reply:
Sorry, I wanted to write "manually impl" instead of "derive".
I think `InterEntities` is good.